bundles/net.tourbook/src/net/tourbook/nutrition/DialogCustomTourNutritionProduct.java

```diff
@@ -185,6 +230,10 @@ private void createUI(final Composite parent) {
                   .hint(_pc.convertWidthInCharsToPixels(5), SWT.DEFAULT)
                   .align(SWT.BEGINNING, SWT.CENTER)
                   .applyTo(_spinnerBeverageQuantity);
+            if (_isEditMode) {
+               _spinnerBeverageQuantity.setEnabled(_product.isBeverage());
```

Comment:
I'm putting all control enablements into one place, e.g. `enableControls()` which makes it also easier to find issues

Reply:
Fixed